Britannia/localization/english/BTWK_buildings_l_english.yml

```diff
@@ -141,3 +157,31 @@
  building_antonine_wall_01:0 "$building_type_antonine_wall_01$"
  building_antonine_wall_01_desc:0 "$building_type_antonine_wall_01_desc$"
+
+ ################
+ # BATH'S BATHS #
+ ################
+ building_type_btwk_aquae_sulis_baths_01:0 "Roman Baths"
+ building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans built a series of baths around some natural springs from which the local Britons worshipped their God #italic Sulis #!(hence, the name of the settlment #italic Aquae Sulis#!). Over the following centuries during Roman occupation, this settlement would grow into a large resort complex, of which its baths being a staple feature."
+
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths"
+ building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nThe departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms, rather than to the upkeep of amenities like this spa. As such, the once magnificent resort has fallen into disuse and decay, and stands more as a monument to an age long past."
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths - Restored"
+ building_btwk_aquae_sulis_baths_02_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nRenovations have been carried out on the spa resort to allow for the recreational bathing the Romans once enjoyed. This has encouraged a larger influx of freemen to bring their custom to the resort - a great boon to the local economy."
+
+
+ ###################
+ # GLASTONBURY TOR #
+ ###################
+ building_type_btwk_glastonbury_tor_01:0 "Glastonbury Tor"
+ building_type_btwk_glastonbury_tor_01_desc:0 "The pre-Christian history of the tor is quite unknown. Archeological evidence indicates there was no permanent settlement until after around the 6th century AD, and would later give way to Glastonbury Abbey. Arthurian legend, however, suggests this as being the homestead of the mythical druid Merlin..."
+ building_btwk_glastonbury_tor_01:0 "$building_type_btwk_glastonbury_tor_01$"
+ building_btwk_glastonbury_tor_01_desc:0 "$building_type_btwk_glastonbury_tor_01_desc$"
+
+ ##################
+ # DUMBARTON ROCK #
+ ################## 
+ building_type_btwk_dumbarton_rock_01:0 "The Rock of Dumbarton"
+ building_type_btwk_dumbarton_rock_01_desc:0 "#italic Dumbarton Rock#! is a rocky outcrop sitting on the edge of the River Clyde overlooking the settlement of Dumbarton, providing ample defence for the fort sitting atop it. It is commonly associated with being the primary seat of the medieval Kingdom of Strathclyde and is even rumoured to have hosted the legendary wizard Merlin at one time..."
```

Comment:
I would lean into the defensive aspect much more - Dumbarton Rock was said to be one of the strongest fortresses in all of Britannia (which was probably a big reason why Strathclyde survived so long, they just couldn't really fully conquer it)
I would use Alt Clut instead of Strathclyde as that's how the kingdom is localized ingame
Also not sure about "hosted the legendary wizard Merlin" - besides the fact that the tradition isn't really that relevant to why Dumbarton Rock is important, Merlin is not exactly a legendary wizard when he is a playable character in the mod...

Reply:
Hmm, I guess there's a couple of things here:
- In the literature there's an increasing tendency to use Alt Clut to describe the kingdom pre-870 and Ystrad Clud/Strathclyde post-870 (when Dumbarton was sacked and the kingdom seems to have reorganized itself around Govan), so you could say that Alt Clut *is* the English name for most of the time period
- More importantly, it's confusing for players who probably won't make the connection that Alt Clut and Strathclyde are the same thing (though for this you could say something like "Kingdom of Strathclyde, also known as Alt Clut")